nodemongo/index.js

```diff
@@ -0,0 +1,21 @@
+var express = require('express');
+var bodyParser = require('body-parser');
+var multer = require('multer');
+var upload = multer();
+var cookieParser = require('cookie-parser');
```

Comment:
@psinha20 Is multer and cookie-parser package is necessary here? or you did copy-paste the code from the previous assignment?